common/apitypes/txl2_test.go

```diff
@@ -0,0 +1,21 @@
+package apitypes
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMarshall(t *testing.T) {
+	var l2tx TxL2
+	_, err := l2tx.MarshalJSON()
+	assert.NoError(t, err)
+}
+
+func TestUnmarshall(t *testing.T) {
+	var l2tx, dest TxL2
+	bytes, err := l2tx.MarshalJSON()
+	assert.NoError(t, err)
+	err = dest.UnmarshalJSON(bytes)
+	assert.NoError(t, err)
+}
```

Comment:
Suggestion: combine this two tests in a single one that:

1. Initialize a `TxL2` with some random data
2. Marshals the tx
3.  Unmarshals the tx
4. Assert that the unmarshaled tx has the same values as the marshaled one